apps/settings/js/panels/achievements/panel.js

```diff
@@ -0,0 +1,27 @@
+define(function(require) {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+  var AchievementsList = require('panels/achievements/achievements_list');
+  var SettingsListener = require('shared/settings_listener');
+
+  return function ctor_achievements_panel() {
+    var achievementsList;
+    var update = achievements => achievementsList.achievements = achievements;
+
+    return SettingsPanel({
+      onInit: function(panel) {
+        achievementsList = AchievementsList(panel);
+      },
+
+      onBeforeShow: function() {
+        SettingsListener.observe(
+          'achievements', achievementsList.achievements, update);
```

Comment:
If `achievementsList.achievements` is used as the default value then you are going to set an `ObservableArray` type property to an array (`achievementsList.achievements` = `achievementsList.achievements`).